chef/cookbooks/ceph/recipes/glance.rb

```diff
@@ -19,6 +19,7 @@
   mode 0640
 end
 
-execute "create new pool #{glance_pool}" do
-  command "ceph osd pool create #{glance_pool} 64"
+ceph_pool glance_pool do
+  pool_name glance_pool
+  pg_num 128
```

Comment:
My ceph knowledge isn't that good, so maybe my question is stupid. But why the change from 64 to 128? Was that indented or just an accident?

If it was indented, mentioning the reasons it the commit message would be good. In general I think it is a good pratice to have separate commits for refactoring (the change from the execute resource to the ceph_pool resource) and functional changes (changing the pg_num default).